doc/api/crypto.md

```diff
@@ -3840,7 +3840,7 @@ passed to [`crypto.createPrivateKey()`][]. If it is an object, the following
 additional properties can be passed:
 
 * `dsaEncoding` {string} For DSA and ECDSA, this option specifies the
-  format of the generated signature. It can be one of the following:
+  format of the signature. It can be one of the following:
```

Comment:
This one should also be reverted since it performs a signing operation.

```suggestion
  format of the generated signature. It can be one of the following:
```